airflow/contrib/operators/gcf_function_delete_operator.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import logging
+import re
+
+from googleapiclient.errors import HttpError
+
+from airflow.contrib.hooks.gcf_hook import GCFHook
+from airflow.models import BaseOperator
+from airflow.utils.decorators import apply_defaults
+
+logger = logging.getLogger(__name__)
```

Comment:
https://github.com/apache/incubator-airflow/blob/master/airflow/contrib/operators/dataproc_operator.py#L242 use the BaseOperator's mixed in logger.